src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -1183,6 +1183,121 @@ public with sharing class ADDR_Addresses_TEST {
 
     /*********************************************************************************************************
     @description
+        Add an existing Contact to a Household. 
+        Pre-populate the Contact's address with the Household Address
+        in the same way Manage Household pre-populates Contact's Mailing Address with Account's Billing Address.  
+    verify:
+        The Contact's current Address is equal to the new Household Address.
+        Contact's "Primary Address Type" is the same as the Address' "Address Type".
+    **********************************************************************************************************/        
+    static testMethod void addExistingContactToNewHouseholdShouldSyncPrimaryAddressTypeWithAddressType() {
+        if (strTestOnly != '*' && strTestOnly != 'addExistingContactToNewHouseholdShouldSyncPrimaryAddressTypeWithAddressType') return;
+        
+        UTIL_CustomSettingsFacade.getContactsSettingsForTests(
+            new npe01__Contacts_and_Orgs_Settings__c (
+                npe01__Account_Processor__c = CAO_Constants.HH_ACCOUNT_PROCESSOR,
+                Simple_Address_Change_Treated_as_Update__c = true
+            )
+        );
+
+        Integer hhSize = 1;
+        Integer contactSize = 1; 
+        createHHTestData(hhSize, contactSize);
+
+        Account account = new Account(Name = 'AccountWithoutAddress');
+        insert account;
+        Contact contact = new Contact(LastName = 'TestPrimaryAddressTypeExistingContact', AccountId = account.Id);
+        insert contact;
+
+        Test.startTest();
+
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+
+        contact.accountId = listAccT[0].Id;
+        update contact = populateContactAddress(contact, listAddrT[0]);
```

Comment:
I understand what it does, but it feels weird to do an update call and a variable assignment on the same line. It also feels a bit strange to reassign the contact variable to the same underlying contact, like we're passing around pointers for no reason. Would you mind putting the update on its own line, and give populateContactAddress a void return type? I'll merge this PR after that. Great work!